tests/o2a_libs/test_el_basic_functions.py

```diff
@@ -79,3 +81,15 @@ def test_trim(self, src_str, expected):
     @parameterized.expand([(" ", "%20"), ("?", "%3F"), ("", ""), (None, "")])
     def test_urlencode(self, src_str, expected):
         self.assertEqual(expected, url_encode(src_str))
+
+    def test_reverse_map(self):
+        props = {"key1": ["value1", "value2"], "key2": ["value3"]}
+        keys = reduce(lambda a, b: a + b, props.values())
```

Comment:
```suggestion
        keys = reduce(lambda a, b: a + b, props.values())
```
List comprehension is better idea. We discussed this topic with food in Ayola ;-)

Reply:
In this case the `reduce` works like `flatmap` and I am not sure how this can be done with single list comprehension?

Reply:
```pytcon
>>> f = {'A': [1,2,3], "B": [4,5,6]}
>>> [item for sublist in f.values() for item in sublist]
[1, 2, 3, 4, 5, 6]
```

Reply:
Or with sum instead of reduce:
```
>>> f = {"A": [1,2,3,4], "B": [5,6,7]}
>>> sum(f.values(), [])
[1, 2, 3, 4, 5, 6, 7]
```